rendercv/templates/classic/cv/components

```diff
@@ -0,0 +1 @@
+../components
```

Comment:
I was not sure about relative paths resolution, so just added a symlink for now.